proto/nym/vpn.proto

```diff
@@ -681,7 +723,7 @@ service NymVpnd {
 
   // Query the account state, which refers to the server side account, as it is
   // known and interpreted by nym-vpnd
-  rpc GetLocalAccountState (GetLocalAccountStateRequest) returns (GetLocalAccountStateResponse) {}
+  rpc GetAccountState (GetAccountStateRequest) returns (GetAccountStateResponse) {}
```

Comment:
Seems to me this should be exposed as rpc **stream**, like daemon pushing to clients the account state whenever it's updated

Reply:
Ideally you want both. And I think the idea is to fold that into the existing streams later on